### DISCUSSION TITLE
What's the WagmiV2 plan?

### DISCUSSION THREAD

**Discussion by Drewsapple** [family/connectkit#320] [General]
Aside from #312, what work needs to be done to support Wagmi V2? It's in beta now, and I'd like to start planning my dapp's migration, including ConnectKit. Is there anything that could be aided by external contributors to get V2 support ready?

**Comment by lochie** [comment]
We're working closely with the wagmi team to plan a release schedule. Still all a work in progress, along with EIP-6963 which should be our next release.

**Reply by RedBeardEth** [reply]
Now the wagmi/viem v2 has been released - do you have an ETA for next release?

**Reply by lochie** [reply]
Working on it 🙌
PR here if you wish to follow the development: https://github.com/family/connectkit/pull/332
